contracts/src/express-relay/ExpressRelay.sol

```diff
@@ -77,6 +77,13 @@ contract ExpressRelay is Helpers, State, ExpressRelayEvents, ReentrancyGuard {
                 multicallStatuses[i].multicallRevertReason = reason;
             }
 
+            if (
+                !multicallStatuses[i].externalSuccess &&
+                multicallData[i].revertOnFailure
+            ) {
+                revert ExternalCallFailed(multicallStatuses[i]);
```

Comment:
nit: feels slightly more readable to place this revert in the `callWithBid` call